web/src/app/dialogs/FormDialog.js

```diff
@@ -99,21 +100,23 @@ export default class FormDialog extends React.PureComponent {
       alert,
       classes,
       confirm,
-      disableGutters,
-      errors,
       isUnmounting,
-      loading,
-      primaryActionLabel, // remove from dialogProps spread
       maxWidth,
       onClose,
       onSubmit,
       subTitle, // can't be used in dialogProps spread
```

Comment:
given the comment, is this something being used or can we remove here and add to the omit?